Makefile

```
@@ -148,7 +148,7 @@ local: generate-secrets
 	fi
 	$(MAKE) set-files-owner SRC=$(CURDIR)/codebase ENVIRONMENT=local
 	docker-compose up -d --remove-orphans
-	docker-compose exec -T drupal with-contenv bash -lc 'composer install; chown -R nginx:nginx .'
+	docker-compose exec -T drupal with-contenv bash -lc "su nginx -s /bin/bash -c 'composer install'; chown -R nginx:nginx ."
```

```suggestion
	docker-compose exec --user nginx -T drupal with-contenv bash -lc "composer install"
	docker-compose exec -T drupal with-contenv bash -lc "chown -R nginx:nginx ."
```
Using `--user` can improve clarity of intent here - eg folks don't have to check where the single quotes are to know that the `chown` runs as root here.

---

Good suggestion. I'll test this suggestion as soon as I can and see. I like your suggestion, it's cleaner than the original.